sqlite3.h not found

I made an attempt to use your wrapper. I added your repository as a git submodule in a `vendor` directory.

I followed the steps in your readme; although I made a minor change in the following line as it appears the path of `sqlite.c` has changed.
```
// Before
sqlite.addCSourceFile("vendor/zig-sqlite/sqlite3.c", &[_][]const u8{"-std=c99"});

// After
sqlite.addCSourceFile("vendor/zig-sqlite/c/sqlite3.c", &[_][]const u8{"-std=c99"});
```

The project would compile fine but would fail at runtime due to missing `sqlite.h`. Not being a zig build expert by any means, I also included the `sqlite.h` in my `build.zig`. Now, it fails at build-time. Here is the my build file:
```
const std = @import("std");

pub fn build(b: *std.build.Builder) void {
    // Standard target options allows the person running `zig build` to choose
    // what target to build for. Here we do not override the defaults, which
    // means any target is allowed, and the default is native. Other options
    // for restricting supported target set are available.
    var target = b.standardTargetOptions(.{});

    // Standard release options allow the person running `zig build` to select
    // between Debug, ReleaseSafe, ReleaseFast, and ReleaseSmall.
    const mode = b.standardReleaseOptions();

    const exe = b.addExecutable("zig-http", "src/main.zig");

    // Build sqlite
    // https://github.com/vrischmann/zig-sqlite
    const sqlite = b.addStaticLibrary("sqlite", null);
    sqlite.addCSourceFile("vendor/zig-sqlite/c/sqlite3.h", &[_][]const u8{"-std=c99"});
    sqlite.addCSourceFile("vendor/zig-sqlite/c/sqlite3.c", &[_][]const u8{"-std=c99"});
    sqlite.linkLibC();
    exe.linkLibC();
    exe.linkLibrary(sqlite);
    exe.addPackage(.{ .name = "sqlite", .path = "vendor/zig-sqlite/sqlite.zig" });
    target.setGnuLibCVersion(2, 28, 0);

    exe.setTarget(target);
    exe.setBuildMode(mode);
    exe.install();

    const run_cmd = exe.run();
    run_cmd.step.dependOn(b.getInstallStep());
    if (b.args) |args| {
        run_cmd.addArgs(args);
    }

    const run_step = b.step("run", "Run the app");
    run_step.dependOn(&run_cmd.step);
}

```

When running `zig build`, I get the following error:

```
$ zig build
./vendor/zig-sqlite/sqlite.zig:8:15: error: C import failed
pub const c = @cImport({
              ^
./zig-cache/o/14e06b957c8c350ff835e95d31fd2640/cimport.h:1:10: note: 'sqlite3.h' file not found
#include <sqlite3.h>
         ^
./vendor/zig-sqlite/sqlite.zig:298:10: note: referenced here
    db: *c.sqlite3,
         ^
./src/main.zig:6:27: note: referenced here
    var db = try sqlite.Db.init(.{
                          ^
zig-http...The following command exited with error code 1:
/usr/bin/zig build-exe /home/kurtis/Code/sandbox/zig-http/src/main.zig -lc /home/kurtis/Code/sandbox/zig-http/zig-cache/o/cc98d7cbd6b618a77ecb081e2f6f160e/libsqlite.a --cache-dir /home/kurtis/Code/sandbox/zig-http/zig-cache --global-cache-dir /home/kurtis/.cache/zig --name zig-http -target native-native-gnu.2.28 -mcpu znver2 --pkg-begin sqlite /home/kurtis/Code/sandbox/zig-http/vendor/zig-sqlite/sqlite.zig --pkg-end --enable-cache 
error: the following build command failed with exit code 1:
/home/kurtis/Code/sandbox/zig-http/zig-cache/o/1a19e351c473280c46346cace1ff8995/build /usr/bin/zig /home/kurtis/Code/sandbox/zig-http /home/kurtis/Code/sandbox/zig-http/zig-cache /home/kurtis/.cache/zig
```

This is my directory structure (including your repository):
```
.
├── build.zig
├── src
│   ├── main.zig
├── vendor
│   └── zig-sqlite
│       ├── build.zig
│       ├── c
│       │   ├── sqlite3.c
│       │   └── sqlite3.h
│       ├── errors.zig
│       ├── gyro.zzz
│       ├── LICENSE
│       ├── query.zig
│       ├── README.md
│       ├── sqlite.zig
│       ├── tools
│       │   ├── Dockerfile.debian
│       │   └── Dockerfile.fedora
│       ├── zig.mod
│       └── zigmod.lock

```

Here is my simple `main.zig` source file:
```
const std = @import("std");
const sqlite = @import("sqlite");

pub fn main() anyerror!void {

    var db = try sqlite.Db.init(.{
        .mode = sqlite.Db.Mode{ .File="sqlite.db" },
        .open_flags = .{
            .write = true,
            .create = true,
        },
        .threading_mode = .MultiThread,
    });

    

    std.log.info("All your codebase are belong to us.", .{});
}
```

I am using Zig version `0.8.1` on Manjaro Linux (x86_64)

From the error, I believe that the issue is related to the build but I honestly don't know enough about the build system yet to debug it. My hunch is that it's related to the path specified here in `sqlite.zig`:
```
pub const c = @cImport({
    @cInclude("sqlite3.h");
});
```

Reading [this old Github Issue](https://github.com/ziglang/zig/issues/1040) suggests that `@cInclude` only searches system paths.

Do you have any suggestions? Worst-case, I can install the sqlite header system-wide but it'd be a better experience to simply vendor it in like this.

Thanks!

--- a/README.md
+++ b/README.md
@@ -72,7 +72,8 @@ If you want to use the bundled sqlite source code file, first you need to add it
 
 ```zig
 const sqlite = b.addStaticLibrary("sqlite", null);
-sqlite.addCSourceFile("third_party/zig-sqlite/sqlite3.c", &[_][]const u8{"-std=c99"});
+sqlite.addCSourceFile("third_party/zig-sqlite/c/sqlite3.c", &[_][]const u8{"-std=c99"});
+sqlite.addIncludeDir("third_party/zig-sqlite/c");
 sqlite.linkLibC();
 ```
 

